src/main/java/io/gingersnapproject/airports/client/GingersnapAPIClient.java

```diff
@@ -15,7 +17,7 @@ public interface GingersnapAPIClient {
    @Path("/us-flight/{code}")
    CacheFlight flight(@PathParam("code") String code);
 
-//   @GET
-//   @Path("/us-flight/{code}")
-//   List<FlightDTO> departures(Integer day);
+   @GET
+   @Path("/dashboard-departure/{code}")
+   DashboardFlightDTO departure(@PathParam("code") String code);
```

Comment:
I guess we'll also need to expose just `/dashboard-departure/` so that we can get all the keys in the first place.